distribution/docs/src/main/resources/content/_running/directory-monitor-ingest.adoc

```diff
@@ -0,0 +1,15 @@
+:title: Content Directory Monitor Ingest
+:type: dataManagement
+:status: published
+:summary: Ingesting data through the Content Directory Monitor.
+:parent: Ingesting Data
+:order: 02
+
+== {title}
+
+The ${ddf-catalog} application contains a Content Directory Monitor feature that allows files placed in a single directory to be monitored and ingested automatically.
+For more information about configuring a directory to be monitored, see <<_configuring_the_content_directory_monitor,Configuring the Content Directory Monitor>>.
+
+Simply place the desired files in the monitored directory and it will be ingested automatically.
```

Comment:
"Simply" sounds patronizing. How about  just "Files put into the monitored directory are automatically ingested"

Reply:
I agree. removed.